components/dashboard/src/components/ConfirmationModal.tsx

```diff
@@ -45,7 +45,7 @@ export default function ConfirmationModal(props: {
     }
 
     const buttons = [
-        <button className="secondary" onClick={props.onClose}>Cancel</button>,
+        <button className="secondary" onClick={props.onClose} autoFocus>Cancel</button>,
```

Comment:
**issue**: This still removes the project when pressing enter. 🤷

Reply:
Resolved with dcf0b6d. See https://github.com/gitpod-io/gitpod/pull/7275#issuecomment-997986614.